.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,6 @@
+Please include the following information when opening an issue:
```

Comment:
It just copies the text directly to new issues and PRs. I don't feel the need to prescribe a rigid format, but I'll happily merge anything you suggest.